rsearch/bin/main.go

```diff
@@ -0,0 +1,57 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	search "github.com/romana/contrib/rsearch"
```

Comment:
Historical reasons. Also i don't see any harm in it - does anyone feel like this need to be reverted back to package name ?

Also, does anyone have ideas for better package name ?